tools/preview/preview.js

```diff
@@ -44,41 +44,42 @@ import './global.js';
 // Rendered as controls; these properties are assigned
 //      to the document root element
 // @todo: resolve errors on 'name' and 'title' in console
-export const globalTypes = {
```

Comment:
Was this not working? I'm wondering why restructure this? I pulled these out into separate exports for clarity and reusability for other projects who might want to extend part of our setup.

Reply:
Ok as per the docs, preview.js needs a default export.
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#default-export-in-previewjs

Reply:
Ah, yes I see. Maybe a middle ground is to keep the distinct exports but add:

```
export default {
    globalTypes,
    argTypes,
     args,
     parameters,
     decorators,
}
```